components/adblock_rust_ffi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Brian R. Bondy <netzen@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-adblock = { version = "0.7.3", default-features = false, features = ["full-regex-handling", "debug-info", "css-validation"] }
+adblock = { version = "0.7.4", default-features = false, features = ["full-regex-handling", "debug-info", "css-validation"] }
```

Comment:
Primarily https://github.com/brave/adblock-rust/pull/273, along with a dependabot update for a dev-dependency and two small cleanup commits.

Reply:
added https://github.com/brave/adblock-rust/commit/9871e4b8989376ee7c6c185623251f34c6cbf75f as well